docs/local_loadtest.md

```diff
@@ -73,8 +73,26 @@ Using a different terminal it's possible to issue commands to create and run a l
 
 ### Create a new load-test agent
 
+To start a new load-test agent via API, the request structure should be in the form of:
+
+```json
+{
+  "LoadTestConfig": {
+    ...
+  },
+  "SimpleControllerConfig": {
+    ...
+  },
+  "SimulControllerConfig": {
+    ...
+  }
+}
+```
+
+The API will create controller specific configuration from the request. However, if there is no controller configuration or errors while reading it from the request, the server will read the default controller configuration locally.
+
 ```sh
-curl -d @config/config.json http://localhost:4000/loadagent/create?id=lt0
+curl -d @config/data.json http://localhost:4000/loadagent/create?id=lt0
```

Comment:
As discussed offline, let's give the user an easy way (one liner possibly) to generate this data from a local config file.